docs/dev/development.md

```diff
@@ -71,7 +71,7 @@ This remote can be used to merge changes from Cirq's main repository into your l
     ```
    
     You can check the branches that are on the ```upstream``` remote by
-    running ```git remote -va``` or ```git branch -r```.
+    running ```git remote -v``` or ```git branch -r```.
```

Comment:
Given the context this might have been - @dstrain115, #736
```suggestion
    running `git branch -va` or `git branch -r`.
```
(Also the triple backticks are redundant for inline code-like-format.)

Reply:
That suggestion makes sense. I've applied it.